api/CampusService.pb.go

```diff
@@ -1957,6 +1957,651 @@ func (x *Cafeteria) GetLatitude() float64 {
+		mi := &file_CampusService_proto_msgTypes[30]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *ResponsiblePersonElement) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*ResponsiblePersonElement) ProtoMessage() {}
+
+func (x *ResponsiblePersonElement) ProtoReflect() protoreflect.Message {
+	mi := &file_CampusService_proto_msgTypes[30]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use ResponsiblePersonElement.ProtoReflect.Descriptor instead.
+func (*ResponsiblePersonElement) Descriptor() ([]byte, []int) {
+	return file_CampusService_proto_rawDescGZIP(), []int{30}
+}
+
+func (x *ResponsiblePersonElement) GetName() string {
+	if x != nil {
+		return x.Name
+	}
+	return ""
+}
+
+func (x *ResponsiblePersonElement) GetTelephone() string {
+	if x != nil {
+		return x.Telephone
+	}
+	return ""
+}
+
+func (x *ResponsiblePersonElement) GetEmail() string {
+	if x != nil {
+		return x.Email
+	}
+	return ""
+}
+
+func (x *ResponsiblePersonElement) GetFaculty() string {
+	if x != nil {
+		return x.Faculty
+	}
+	return ""
+}
+
+func (x *ResponsiblePersonElement) GetTumID() string {
+	if x != nil {
+		return x.TumID
+	}
+	return ""
+}
+
+type GetBuilding2GpsResponse struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Building2Gps []*Building2GpsElement `protobuf:"bytes,1,rep,name=building2Gps,proto3" json:"building2Gps,omitempty"`
+}
+
+func (x *GetBuilding2GpsResponse) Reset() {
+	*x = GetBuilding2GpsResponse{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_CampusService_proto_msgTypes[31]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *GetBuilding2GpsResponse) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*GetBuilding2GpsResponse) ProtoMessage() {}
+
+func (x *GetBuilding2GpsResponse) ProtoReflect() protoreflect.Message {
+	mi := &file_CampusService_proto_msgTypes[31]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use GetBuilding2GpsResponse.ProtoReflect.Descriptor instead.
+func (*GetBuilding2GpsResponse) Descriptor() ([]byte, []int) {
+	return file_CampusService_proto_rawDescGZIP(), []int{31}
+}
+
+func (x *GetBuilding2GpsResponse) GetBuilding2Gps() []*Building2GpsElement {
+	if x != nil {
+		return x.Building2Gps
+	}
+	return nil
+}
+
+type Building2GpsElement struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Id        string `protobuf:"bytes,1,opt,name=id,proto3" json:"id,omitempty"`
```

Comment:
Since I have no idea, how the building ids look like (I dont have access to the database):
- What is the number of Buildings in this relationship?
- Could you send them to me and would adding another field for the new (NavigaTUM) ID be possible (assuming that our IDs are not 1:1 compatible?

The reason for this comment is that we identified this as a blocking issue in https://github.com/TUM-Dev/Campus-Android/pull/1462

Since the `RoomId`, `RoomCode`, `BuildingNr` and `ArchId` is already present, this should not be a huge issue, but I would need some more information about what field does mean what..

Reply:
can only give an overview over that type: The building_nr has the type varchar with length 8, but I don't know how the ids look like in the database. The database scheme itself is published in the [db-models-branch](https://raw.githubusercontent.com/TUM-Dev/Campus-Backend/db-models/model/souce-schema.sql) in this Repo. 

I can add this new field in this implementation, how would it look like? Is it a string or an Integer?

Reply:
Example values which I would expect (from similar-ish naming):
```
ArchId: "27543"
RoomCode: "5407.01.760D"
BuildingNr: "5407"
RoomId: internal Id; just used in this API
```

Is this assumption right?

Reply:
From my point of view, at least the datatypes would match this assumption. @joschahenningsen ca you provide any example data?